dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_dashboard.feature

```diff
@@ -32,6 +33,15 @@ Feature: Using the teacher dashboard
     Then I navigate to teacher dashboard for the section I saved
     And I wait until element "#uitest-course-dropdown" is visible
     And I select the "All the Things! *" option in dropdown "uitest-course-dropdown"
+    Then I append "/?enableExperiments=section_progress_v2" to the URL
```

Comment:
What do you think about moving this to line 29 before the v1 progress tab tests to test that it shows up even with the toggle button displayed?

Reply:
This was tricky... I could not get the experiment to work unless I added this experiment ON the progress view... I still don't know why that was.  For that reason, I added the experiment right before it was needed.  BUT I think we could move it to as soon as the test got to the progress page.